app/controllers/admin/conferences_controller.rb

```diff
@@ -61,11 +61,19 @@ def index
 
     def new
       @conference = Conference.new
+      @organizations = {}
+      Organization.all.each do |organization|
+        @organizations.store(organization.name, organization.id) if can? :create, Conference.new(organization: organization)
+      end
     end
 
     def create
-      @conference = Conference.new(conference_params)
-
+      conference_params_copy = conference_params
```

Comment:
This will not really "copy" the parameters. Though, I'm not clear why you would need to do it at all.